### PR TITLE
PATCH RELEASE only download files not on S3 + FHIR server

### PR DESCRIPTION
**Pull request by leite08**
Ref. metriport/metriport-internal#785

### Description

Only download files not on S3 + FHIR server

### Release Plan

- ⚠️ This is pointing to `master`

### CODE REVIEW

File: api/app/src/external/commonwell/document/document-query.ts [hunk 1]
```diff
@@ -400,17 +445,14 @@ export async function downloadDocsAndUpsertFHIR({
...
                 const url = new URL(signedUrl);
                 const s3Location = url.origin + url.pathname;
                 return {
-                  bucket: s3BucketName,
-                  key: s3FileName,
+                  bucket: fileInfo.fileLocation,
+                  key: fileInfo.fileName,
                   location: s3Location,
-                  size: existingSize,
+                  size: fileInfo.fielSize,
```
Comment (leite08) [inline api/app/src/external/commonwell/document/document-query.ts:455]: typo

File: api/app/src/command/medical/document/document-query-storage-info.ts [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+          message: String(error),
+          docId: doc.docId,
+          fhirDocId: doc.fhirDocId,
+        });
+        throw error;
+      }
+    })
+  );
+  if (errors.length > 0) {
+    capture.message(`Errors getting info from S3`, {
```
Comment (leite08) [inline api/app/src/command/medical/document/document-query-storage-info.ts:70]: log as well

**Review by Goncharo — APPROVED** [review]
:shipit: - one nit comment

File: api/app/src/command/medical/document/document-query-storage-info.ts [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+};
+
+type SimpleFile = {
+  fhirDocId: string;
+  docId: string;
+  fileName: string;
+  fileLocation: string;
+};
+
+// TODO convert from Document to a Metriport shape, not Commonwell's
```
[the remark this answers is not in the document]
Comment (leite08) [reply api/app/src/command/medical/document/document-query-storage-info.ts]: I was referring to the input `Document`, updated the comment to better represent that 👍

**Pull request merged**